### ISSUE TITLE
White line around images on IE11

### ISSUE THREAD

**Issue by frontero** [karlhorky/gray#77]
The script works fine except on IE11. If images are next to each other it shows a white line around the image after the script has loaded.  Example here: 
https://www.burobenk.nl/het-team-van-benk

**Comment by karlhorky** [comment]
Hi there, thanks for the issue.

I tried opening the site in Chrome first, and I also saw borders between the images. 

<img width="1038" alt="screen shot 2017-10-17 at 16 55 40" src="https://user-images.githubusercontent.com/1935696/31672172-14a34e6a-b35c-11e7-8066-8a87217beb30.png">

Upon closer inspection, I see that the columns are being assigned half pixel widths (at least on my 13" MacBook Pro).

<img width="329" alt="screen shot 2017-10-17 at 16 56 32" src="https://user-images.githubusercontent.com/1935696/31672235-4c2094ce-b35c-11e7-8eb0-2b40075c9565.png">

Half-pixel widths can lead to weirdness and cross-browser inconsistencies. This may also be the problem, as opposed to there being an issue with the IE polyfill.

Does the issue go away if you have a container that can be cleanly divided by 4 without half-pixel width columns?
